package.json

```diff
@@ -5,16 +5,19 @@
   "main": "index.js",
   "scripts": {
     "build": "rm -rf dist && webpack",
-    "test": "echo \"Error: no test specified\" && exit 1"
+    "test": "rm -rf dist && tsc && node dist/test/index.js"
```

Comment:
You can actually refer to other npm scripts inside of other npm scripts, so: `npm run build && node dist/test/index.js` here.